packages/palette/src/elements/Image/LazyImage.tsx

```diff
@@ -73,11 +75,23 @@ export const LazyImage: React.FC<LazyImageProps> = ({
 
   const handleLoad = () => setImageLoaded(true)
 
+  // If there is a blurPlaceholder, use a regular Box to avoid a grey background.
+  const Wrapper = blurDataURL ? Box : SkeletonBox
```

Comment:
**praise:** Aha, so this is the reason your latest video no longer shows that thrashing of _pulsing skeleton → blurhash_, yes? I like it!

Reply:
Yea, this and the `display: block` was necessary to get no initial grey box, and also for the loaded image to perfectly 'hide' the background'ed div.

I fiddled with taking this code and just including it in Force (as part of `GridItem` and/or `MagnifyImage`) and couldn't quite get things to render cleanly. This approach in Palette doesn't seem particularly objectionable and it's opt-in where the client needs to pass a truth-y `blurDataURL` prop, so I feel good about including it. At the end of the day, it's just some conditional styling.

Plus, doing this in Palette means it should be reusable other places in Force (besides `Grid/MagnifyImage`) where we may want this behavior, like in artwork rails or elsewhere.